plutus-scb/src/Plutus/SCB/Monitoring.hs

```diff
@@ -65,8 +64,6 @@ defaultConfig = do
                         , scMinSev = minBound
                         , scMaxSev = maxBound
                         }]
-  let observables = (Just $ ObservableTraceSelf [MonotonicClock, MemoryStats])
-  CM.setSubTrace c "processAllContractOutboxes" observables
```

Comment:
?

Reply:
The two lines meant that the `MonotonicClock` and `MemoryStats` observables would be measured every time `processAllContractOutboxes` ran, resulting in 2 lines of log output every two seconds.

This can all be enabled with a config file as well. I just wanted to make the default logging configuration a bit less verbose.